.babelrc

```diff
@@ -0,0 +1,28 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "modules": false,
+        "useBuiltIns": "entry",
+        "corejs": 3,
+        "shippedProposals": true
```

Comment:
This was recently added and replaces `@babel/plugin-proposal-class-properties`, see [shippedProposals](https://babeljs.io/docs/en/babel-preset-env#shippedproposals)

Without this I saw duplicate plugin errors with fresh installs, presumably because it now ships in `preset-env`